src/app/pages/SearchResultsPage/ResultsElsewhere.tsx

```diff
@@ -0,0 +1,144 @@
+    .filter(key => !!resultsInScopes[key])
+    .filter(scopeKey => scopeKey !== wantedScopeKey)
+  const otherScopes = otherScopesKeys.map(getScopeForKey)
+  const hasResultsOnDifferentParatimes = otherScopes.some(
+    scope => scope.network === wantedScope.network && scope.layer !== wantedScope.layer,
+  )
+  const hasResultsOnDifferentNetworks = otherScopes.some(scope => scope.network !== wantedScope.network)
+  const hasAllKindsOfResults = hasResultsOnDifferentParatimes && hasResultsOnDifferentNetworks
+  const location = t(
+    hasAllKindsOfResults
+      ? 'search.otherResults.otherNetworksAndParatimes'
+      : hasResultsOnDifferentNetworks
+      ? 'search.otherResults.otherNetworks'
+      : 'search.otherResults.otherParatimes',
+  )
+
+  const totalNumberOfResults = otherScopesKeys.reduce(
+    (partialSum, scopeKey) => partialSum + resultsInScopes[scopeKey],
+    0,
+  )
+
+  // openByDefault={scope.network === Network.mainnet && !hasNoResultsInWantedScope}
+  const openByDefault = false
+
+  const [open, setOpen] = useState(openByDefault)
+
+  const hasMainnetResults = otherScopes.some(scope => scope.network === Network.mainnet)
+
+  const themes = getThemesForNetworks()
+  const networkNames = getNetworkNames(t)
+
+  const notificationTheme = hasMainnetResults ? themes[Network.mainnet] : themes[otherScopes[0].network]
+
+  if (!otherScopes.length) return null
+
+  const resultsInNetworks: Record<Network, number> = {} as any
+  const allNetworks = RouteUtils.getEnabledNetworks()
+  allNetworks.forEach(net => (resultsInNetworks[net] = 0))
+  otherScopes.forEach(scope => (resultsInNetworks[scope.network] += resultsInScopes[scope.key]))
+
+  if (!open) {
+    return (
+      <NotificationBox theme={notificationTheme} onClick={() => setOpen(true)}>
+        <ZoomIn />
+        <span>
+          <Trans
+            t={t}
+            i18nKey={
+              alsoHasWantedResults
+                ? 'search.otherResults.clickToShowMore'
+                : 'search.otherResults.clickToShowThem'
+            }
+            values={{
+              countLabel: t(alsoHasWantedResults ? 'search.results.moreCount' : 'search.results.count', {
+                count: totalNumberOfResults,
+              }),
+              location,
+            }}
+          />
+        </span>
+      </NotificationBox>
+    )
+  }
+
+  return (
+    <>
+      <NotificationBox theme={notificationTheme} onClick={() => setOpen(false)}>
+        <ZoomOut />
+        {t('search.otherResults.clickToHide', { location })}
+      </NotificationBox>
+      {hasResultsOnDifferentParatimes && (
+        <ResultsInNetworkThemed
+          network={wantedScope.network}
+          title={t('search.otherResults.otherParatimesOnNetwork', {
+            network: networkNames[wantedScope.network],
+          })}
+          searchQueries={searchQueries}
+          numberOfResults={resultsInNetworks[wantedScope.network]}
+          roseFiatValue={roseFiatValue}
+        />
+      )}
```

Comment:
- https://1232a66b.oasis-explorer.pages.dev/mainnet/emerald/search?q=0xBA504818FdD8D3dBA2Ef8fD9B4F4D5c71aD1d1D3
- expand more results
- see the same Emerald Mainnet account listed again

![1232a66b oasis-explorer pages dev_mainnet_emerald_search_q=0xBA504818FdD8D3dBA2Ef8fD9B4F4D5c71aD1d1D3](https://github.com/oasisprotocol/explorer/assets/3758846/17b15798-98cf-4135-bbc5-8be5a7616f4b)

Reply:
Just pushed a big simplification and code removal. Should take care of that. too.

Reply:
Yes, now solved.